pkg/minikube/config/profile.go

```diff
@@ -73,6 +74,15 @@ func ProfileExists(name string, miniHome ...string) bool {
 	return err == nil
 }
 
+// CurrentProfile returns the name of the current profile
+func CurrentProfile() string {
+	profile := viper.GetString(MachineProfile)
+	if profile != "" {
+		return profile
+	}
+	return "minikube"
```

Comment:
A lot of trouble has come from API's using a similar function, but needing to access multiple profiles. 

Instead, can we simply update our code to use viper.GetString(MachineProfile) in the `cmd` package, and make sure to pass the correct profile into other packages?

NOTE: I don't think viper.GetString(MachineProfile) should ever be "" unless the user specifies it to shoot themselves in the foot.

Reply:
sounds good, I'll use viper.GetString(MachineProfile) instead.